src/test/java/website/frontrow/util/CollisionHandlerTest.java

```diff
@@ -130,9 +130,9 @@ public void platformMotionUpAABBTest()
                 Cell.WALL, Cell.PLATFORM, Cell.WALL,
                 Cell.WALL, Cell.WALL, Cell.WALL), 3, 3));
         CollisionHandler handler = new CollisionHandler(check);
-        assertFalse(handler.checkLevelAABB(
-                new AABB(new Point(1, 1), new Point(2, 2)), new Point(0, -1)
-        ));
+        Cell cell = handler.checkLevelAABB(
+        		new AABB(new Point(1, 1), new Point(2, 2)), new Point(0, -1));
+        assertTrue(cell.getType() == Cell.EMPTY);
```

Comment:
Use assertEquals instead of == operator combined with assertTrue, == does not work properly for non-objects.
<testing